content/docs/user-guide/contributing/docs.md

```diff
@@ -62,8 +62,8 @@ $ git clone git@github.com:<username>/dvc.org.git
 $ cd dvc.org
 ```
 
-Make sure you have the latest version of [Node.js](https://nodejs.org/en/), and
-install [Yarn](https://yarnpkg.com/):
+Make sure you have a **recent** version of [Node.js](https://nodejs.org/en/),
```

Comment:
what does "recent" mean?

Reply:
Good question. I'm not sure what's the oldest version this works on, probably its safe to recommend 10+ cc @iAdramelk

Reply:
Its easier to revert to "latest" but will always work? We can't guarantee it, maybe we should get specific. But up to you, no strong opinion from my part.

Reply:
yes, I would just specify explicitly the versions we support

Reply:
This is info. I don't currently know exactly. I know the std. node provided by apt-get (Ubuntu) is really old so I had to install [like this](https://github.com/nodesource/distributions/blob/master/README.md#debinstall). @iAdramelk could you chime in here please?

Reply:
we have engines specified as

```
  "engines": {
    "node": "^12.0.0"
  },
``` - in blog (which will be part of the dvc.org soon). @fabiosantoscode and @iAdramelk should have more context - ping them in Slack if needed

Reply:
Pinged Fabian but he's probably offline rn. I'm using 12 for now so we can merge this and I can update with whatever they say in another PR later. Going to focus on other doc PRs for now.